deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-console-stack.ts

```diff
@@ -322,10 +322,8 @@ export class MigrationConsoleStack extends MigrationServiceCore {
                 containerPort: 8000,
                 protocol: Protocol.TCP
             }]
-            serviceDiscoveryPort = 8000
-            serviceDiscoveryEnabled = true
             imageCommand = ['/bin/sh', '-c',
-                '/root/loadServicesFromParameterStore.sh && python3 /root/console_api/manage.py runserver_plus 0.0.0.0:8000'
+                '/root/loadServicesFromParameterStore.sh && python /root/console_api/manage.py runserver_plus 0.0.0.0:8000 --cert-file cert.crt'
```

Comment:
nit: I think I mentioned somewhere else but changing to python here makes me think we could support python2 which I don't think is the case

Reply:
Retested with python3 and it's working so i changed it back. Looks like python is a symlink to python3 anyways